opencv/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: opencv
+  version: 2.4.11
```

Comment:
Would be nice to set this with jinja templates so it is easy to update. Here is a [simple example]( https://github.com/conda-forge/autoconf-feedstock/blob/309fba96ae87fdc3b1eb21f6c45499f63a3925d0/recipe/meta.yaml ).